packages/headings/LinkedHeading.jsx

```diff
@@ -20,15 +21,27 @@ export function LinkedHeading({
   ctaText = "View all",
   ctaLink,
   ctaNewWindow,
+  isAlwaysLight,
```

Comment:
In the Horizontal Video Testimonial Section, since the section heading (`LInkedHeading` package) will always render on top of a dark overlay, the heading text should be always light (white, in our case) for both light and dark mode, so I created this variant.